homeassistant/components/climate/homematicip_cloud.py

```diff
@@ -82,7 +85,13 @@ def current_humidity(self):
     @property
     def current_operation(self):
         """Return current operation ie. automatic or manual."""
-        return HMIP_STATE_TO_HA.get(self._device.controlMode)
+        if self._device.boostMode:
+            return STATE_BOOST
```

Comment:
Only climate states available in the base climate component are allowed to be returned here.